src/lib/openapi/spec/validate-edge-tokens-schema.ts

```diff
@@ -9,12 +9,7 @@ export const validateEdgeTokensSchema = {
     properties: {
         tokens: {
             type: 'array',
-            items: {
-                anyOf: [
-                    { $ref: '#/components/schemas/edgeTokenSchema' },
-                    { type: 'string' },
-                ],
-            },
+            items: { $ref: '#/components/schemas/edgeTokenSchema' },
```

Comment:
Can we get an example and a description going for this property?

Reply:
Added in 6ac9cc0